src/test/java/io/prometheus/wls/rest/ExporterServletTest.java

```diff
@@ -146,6 +148,17 @@ public void onGet_defineConnectionUrlFromContext() throws Exception {
                    equalTo(String.format(URL_PATTERN, LiveConfiguration.WLS_HOST, HttpServletRequestStub.PORT)));
     }
 
+    @Test //@Ignore("FIX ME in ExporterServletTest")
```

Comment:
Should this commented out @Ignore be removed?

Reply:
Yup - thanks for noticing that,